benchmarks/bench_multi_gpu.py

```diff
@@ -14,7 +14,7 @@
 import torch.multiprocessing as mp
 import torch.nn as nn
 import torch.utils.benchmark as benchmark
-from float8_experimental.float8_linear import Float8Linear
+from float8_experimental.float8_linear import Float8Linear, TensorScalingType
```

Comment:
How useful is this benchmark in general?

Reply:
I haven't used it recently